app/services/generate_training_data.rb

```diff
@@ -0,0 +1,48 @@
+require "openai"
+
+class GenerateTrainingData
+  def initialize(import_id)
+    @client = OpenAI::Client.new
+    @import_id = import_id
+  end
+
+  def call
+    response = @client.chat(
+      parameters: {
+        model: "gpt-4",
+        messages: [{role: "user", content: prompt}],
+        temperature: 0.2
+      }
+    )
+
+    response.dig("choices", 0, "message", "content")
+  rescue => e
+    puts("Error: #{e.message}")
+  end
+
+  private
+
+  def extract_text
+    text = PdfToText.new(@import_id).call
+    text.gsub('\n', " ").gsub(/\s+/, " ").strip
+  end
+
+  def base_prompt
+    "Get the work processes from the following text in JSON format. JSON output needs the following fields:\
+      title: title of the work process.
+      description: description for the work process.
+      defaultHours: amount of hours. It is optional. If value not found, return null.
```

Comment:
Can we add more examples where this value should be filled in and adjust the prompt if needed to return the correct value?

Reply:
@jhash I'm not sure I know what you mean by this. `defaultHours` is just a number